community/2024-08-28-community-meeting.mdx

```diff
@@ -17,15 +17,35 @@ title: 'Meeting Agenda - 2024-08-28'
 
 {/* Add summary of topics during the meeting. Each Topic should have an H3 heading. A list is usually sufficient for the recap */}
 
-### Topic 1
-
-- Note 1
-- Note 2
-
-### Topic 2
-
-- Note 1
-- Note 2
+### DEMO: wasmCloud 1.2 / wash 0.31 showcase
+
+- wasmCloud 1.2 is our latest release, now out! Check out this a great blog post on [wasmCloud 1.2](https://wasmcloud.com/blog/2024-08-27-wasmcloud-1.2-improved-streams-automatic-config-updates) which is well worth diving into.
+- Highlights include: automatic config updates for providers (update to the latest SDK - there is a great first issue for implementing this in Go), improved streams, additional policy and secrets validation, better downstream link validation, and fully-tunable wasmtime runtime parameters (important to be able to control how much memory a component can use).
+- In focus: `wasi-blobstore` streaming has seen a whole suite of improvements; manifests are better. Suggest checking out the [blobby example](https://github.com/wasmCloud/wasmCloud/blob/04d7e8e902ec27dcc20e1deeb222574554166ad1/examples/rust/components/blobby/README.md?plain=1#L4) to see how this works in practice.
+- This closes out all the remaining issues in the Q2 roadmap.
+- In wash 0.31 we have a couple of great improvements - as always check out the recording for Brooks's demo.
+- FEATURE: If you output the version for wash we now get the version and the embeded default versions of NATS, wadm and wasmCloud that wash will utilize. This is super useful in the command line - easy to do wash version and get all the versions needed by wash.
+- FEATURE: improvements to the tables that we output in the command line. wash app lists - instead of having many separate fields, we have the description listed below the app and have streamlined the information there.
```

Comment:
```suggestion
- FEATURE: improvements to the tables that we output in the command line. `wash app list` - instead of having many separate fields, we have the description listed below the app and have streamlined the information there.
```